include/openthread/nat64.h

```diff
@@ -94,10 +94,10 @@ typedef struct otIp4Cidr
  */
 typedef struct otNat64Counters
 {
-    uint64_t m4To6Packets; ///< Number of packets translated from IPv4 to IPv6.
-    uint64_t m4To6Bytes;   ///< Sum of size of packets translated from IPv4 to IPv6.
-    uint64_t m6To4Packets; ///< Number of packets translated from IPv6 to IPv4.
-    uint64_t m6To4Bytes;   ///< Sum of size of packets translated from IPv6 to IPv4.
+    int64_t m4To6Packets; ///< Number of packets translated from IPv4 to IPv6.
+    int64_t m4To6Bytes;   ///< Sum of size of packets translated from IPv4 to IPv6.
+    int64_t m6To4Packets; ///< Number of packets translated from IPv6 to IPv4.
+    int64_t m6To4Bytes;   ///< Sum of size of packets translated from IPv6 to IPv4.
```

Comment:
Why use `int64_t` instead of `uint64_t`? The counters are for sure non-negative.
Is there any advantage?

We generally try to minimize changing public APIs and types (to ensure backward compatibility) unless we have a good justification.

Reply:
The idea is following the style guide of https://google.github.io/styleguide/cppguide.html#Integer_Types

Given the actual meaning and use cases, this PR is optional.

As these API are recently merged, I would expect the impact of the API change is minimal.